### DISCUSSION TITLE
more/

### DISCUSSION THREAD

**Discussion by giscus** [SMGoro/issues-json-generator#15] [General]
# more/

关于赞助站点工具 关于友链 SummerGoro  我是谁？ 自称 夏五郎(SummerGoro) ，一个来自互联网的观测者/记录者。目前是一位苦逼的高三狗，会弹点钢琴，在b站当摸鱼UP主，不会敲代码，不过喜欢看番和鼓捣服务器，什么都摸过，什么都不精通的那种家伙，最喜欢的歌手是花谱，为人懒散没有规划性，简单来说，是个傻逼。 最近莫名其妙的入坑了东方，在玩了玩正作中的《绯想天则》和《东方永夜抄》，时

https://blog.smgoro.com/more/

**Comment by LineXic** [comment]
没想到在互联网中探索到了你的博客，我之前也用你的[项目](https://github.com/SMGoro/smgoro.github.io)做官网了不过现在在自己尝试并且优化SEO了

在minebbs也有你的身影我们都在为热爱努力着

**Reply by LineXic** [reply]
[minebbs](https://www.minebbs.com/members/summergoro.2823/)
